### ISSUE TITLE
[Feature request] Allowing esheep to fall off occasionally when walking on top of a window

### ISSUE THREAD

**Issue by CC-Hsu** [Adrianotiger/desktopPet#68]
Hi, Team,

First of all, thank you very much for making esheep alive again!

When I grab a sheep to a window top, I observe that the sheep smartly turn back when walking to the edge of the window.

And when I move the window around, the sheep stick to the window.

If I remember correct, the old sheep sometimes falls off from the top of a window when walking to the "cliff", or when the window is suddenly moved away.

It would be more interesting if this feature can be include.

Thanks again & Best Regards.

**Comment by OliverPets** [comment]
In previous versions, the sheep would not stick to windows. This is a feature i would also like to have. 

Maybe this could be one of the options in the menu to enable/disable this feature?

**Comment by CC-Hsu** [comment]
Hi, Oliver,

That's great!

Thanks for all your team's great work~

**Comment by Adrianotiger** [comment]
Yes, this feature was implemented in the last version. But causes a lot of bugs. Sometimes the sheep disappear completely (invisible or opacity is 0.0, don't know exactly).
But if you have a mate that is still on a window, it is better if it will stay on the same position also if you move the window.

1. I can add a menu to the options window.
2. Sheep falls if she is walking and will stick on the window, if she is not walking (like sleeping)
3. Add a xml option to let decide the developer of the mate*.

I see only this 3 options. 
The easiest to implement is the second one. * In future I would like also add some (global) animation options. So that a sheep can for example interact with the mouse or other sheeps. But I need some time to implement it.

**Comment by OliverPets** [comment]
I think option 3 would be best so we can make pets that stick or don't stick. Maybe have sticking on by default? My sheep all have working gravity effects on them so I use the older version sometimes to see what it looks like :P
